### PR TITLE
Style authorize page to look more like ownCloud

### PR DESCRIPTION
**Pull request by DeepDiver1975**
bildschirmfoto von 2017-07-17 16-03-16

bildschirmfoto von 2017-07-17 21-01-10

bildschirmfoto von 2017-07-17 21-06-28

bildschirmfoto von 2017-07-17 21-07-00

### CODE REVIEW

**Comment by PVince81** [comment]
@felixheidecke please check the design of this page.

I'd say the arrow needs more padding on the right.

**Comment by michaelstingl** [comment]
Why put an arrow on a button?

**Comment by DeepDiver1975** [comment]
> Why put an arrow on a button?
> Merge state

no idea - me not care much ...

**Comment by DeepDiver1975** [comment]
arrow killed

**Comment by DeepDiver1975** [comment]
Remaining pages have been styled now as well

File: templates/switch-user.php [hunk 1]
```diff
@@ -21,20 +21,17 @@
...
+	<form id="form-inline" action="" method="post">
+		<p><b><?php p($l->t('Switch user')); ?></b></p>
+		<br>
+		<p><?php p($l->t('You are logged as %s but the application requested access for user %s.', [$_['current_user'], $_['requested_user']])); ?></p>
+		<br>
+		<a href="<?php p($_['logout_url']); ?>">
+			<button><?php p($l->t('Logout and login as %s', $_['requested_user'])); ?></button>
+		</a>
+		<a href="<?php p($_['back_url']); ?>">
+			<button><?php p($l->t('Cancel')); ?></button>
```
Comment (ogoffart) [inline templates/switch-user.php]: Did we not say we kill the 'cancel' button ?

Comment (DeepDiver1975) [inline templates/switch-user.php]: Maybe I should name it different..... let me think about that

**Comment by DeepDiver1975** [comment]
with the new ce theme this just looks really nice

bildschirmfoto von 2017-07-20 11-24-38

File: templates/switch-user.php (hunk 1 above)
Comment (DeepDiver1975) [inline templates/switch-user.php]: okay - ill kill that one as well

Comment (DeepDiver1975) [inline templates/switch-user.php]: dead

**Comment by ogoffart** [comment]
So the authorization success page no longer have links to go back to the main page and other apps? (But I don't know if it should have it anyway.)

**Comment by DeepDiver1975** [comment]
> So the authorization success page no longer have links to go back to the main page and other apps?

indeed

tome to get some design feedback in here @felixheidecke

**Pull request merged**